.github/workflows/e2e-workflow.yml

```diff
@@ -6,6 +6,9 @@ on:
       git_sha:
         type: string
         required: true
+      suite:
```

Comment:
Please consider renaming variable suite to nodeprovisioner. This improves the readability.